### ISSUE TITLE
Unexpected value 'ClickOutsideModule' Angular 10

### ISSUE THREAD

**Issue by Swiftwork** [arkon/ng-click-outside#70]
Hi,

We receive the same error message as the old #7, but we have the following versions:

Angular: 10.2.3
Typescript 3.9.7
ClickOutside: 7.0.1

We are compiling using ivy and removing the module the compilation works. Maybe Angular 10.2 broke compatibility. Using the directive results in a similar issue.

```
An unhandled exception occurred: Unexpected value 'ClickOutsideDirective in ...' declared by the module 'AppModule in ...'. Please add a @Pipe/@Directive/@Component annotation.
```

**Comment by spider9375** [comment]
Same goes for Angular 11.0.2 Typescript 4.0.5

P.S. In my case it was being used for date picker component. With new packets the date picker closes automatically upon clicking out of it.

**Comment by arkon** [comment]
Probably fixed in v8.0.0.
